samples/drivers/watchdog/src/main.c

```diff
@@ -14,39 +14,12 @@
 
 #define WDT_FEED_TRIES 5
 
-/*
- * To use this sample, either the devicetree's /aliases must have a
- * 'watchdog0' property, or one of the following watchdog compatibles
- * must have an enabled node.
- */
-#if DT_NODE_HAS_STATUS(DT_ALIAS(watchdog0), okay)
-#define WDT_NODE DT_ALIAS(watchdog0)
-#elif DT_HAS_COMPAT_STATUS_OKAY(st_stm32_window_watchdog)
-#define WDT_NODE DT_INST(0, st_stm32_window_watchdog)
-#define WDT_MAX_WINDOW  100U
```

Comment:
This special case still needs to be considered.

Reply:
Oh, I didn’t see that one.. thanks!

Reply:
this is a sympthom that we likely need a watchdog specifier, like we have e.g., for adc or pwm.